feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt

```diff
@@ -218,6 +218,7 @@ private fun SpeakerIcon(
         },
         contentDescription = UserIcon.asString(),
         modifier = modifier
+            .size(TimetableGridItemSizes.speakerHeight)
```

Comment:
When `SpeakerIcon` load very small image, icon image displayed such like a "dot".
This is because `SpeakerIcon` doen't have settings about size.
If set the size of image, it can also be resolved.
※ Almost all user icon is larger than `TimetableGridItemSizes.speakerHeight`, so I use this for `size(...)`

before | after
 :--: | :--: 
<img src="https://github.com/DroidKaigi/conference-app-2023/assets/74723074/4b60eea1-5a9b-4116-89c1-168fa9c50712" width="300" /> | <img src="https://github.com/DroidKaigi/conference-app-2023/assets/74723074/c4e22240-45f9-499c-b12f-28977b2cb742" width="300" />